frontend/src/components/SingleRecipe.js

```diff
@@ -174,12 +245,20 @@ export default function SingleRecipe() {
             ) : (
               ''
             )}
-            <LeftSideText>
-              <Button variant="success" onClick={handleRecomend}>
-                Ajánlanád?
-                <span className="sr-only">Ajánlások</span>
-              </Button>
-            </LeftSideText>
+            <ButtonGroup>
+              <ButtonStyle>
+                <Button variant="success" onClick={handleRecomend}>
+                  Ajánlanád?
+                  <span className="sr-only">Ajánlások</span>
+                </Button>{' '}
+              </ButtonStyle>{' '}
+              <ButtonStyle>
+                <Button variant="success" onClick={PDFGenerator}>
+                  Nyomtatás
+                  <span className="sr-only">Ajánlások</span>
```

Comment:
A nyomtatás gombhoz is kellenek az ajánlások?

Reply:
Törölve!